### DISCUSSION TITLE
v4.3.0

### DISCUSSION THREAD

**Discussion by filiptronicek** [aperta-principium/Interclip#278] [General]
## Features ✨ 
- Added the ability to abort file uploads when they are in progress e82c41947b4dca8fcd9e74cefe3779316c64314c
- Added text uploads aka "Interclip gists" 21b2b538a40a73bc528e61650dd9f9a6e94a8348

## Bugs 🐛 

- Fixed a bug in which a file name would be displayed incorrectly in the UI 97c7453f2adf38bfda801de0e85c14ed416126d5
- Made file uploads more reliable ddbfc51d82c73acca2463425a7c7ef9e230e96df
- Fixed a bug with file names being kept in the UI for failed or cancelled uploads 3eea913b96819a2c9486beea42ffcde591881af0
- Fixed drag n' drop from Firefox's download panel f90f4a967da9b0ef47e19446eb71c3c4e9a09870

## Accessibility ♿ 

- Improve focus for logout link 06a6e7e9c3bede5fd391a49e55fa5248dfd172ba
- Fix focusing on the recent clips dropdown dba119379c49a4bd213e2a85554c87bb6efd64ae
- For file uploads, `<dialog>` is now used 28c23d563825f2f990e439a97c53f98cb2561718
- Fixed contrast for percentage on the progress bar b0eb6b1bb6d35f5937e4e2ef4d7ef8f6b898f9d2
- Add the ability to initiate a file picker for keyboard-only users 7ee83f9b170f2ef4fea3d1d4af6c30ecc42da179
- Fix keyboard actions on the closing button for the settings modal e8f7c5f17d80cb1a83a1cff7c7d01fd1bb73847c

## Performance ⚡️

- Removed all external fonts to reduce Interclip's size d3ce2872e4e9d939d97d9caa8c0ff1e7c55ee63c

<hr /><em>This discussion was created from the release <a href='https://github.com/interclip/interclip/releases/tag/v4.3.0'>v4.3.0</a>.</em>

**Comment by bigint** [comment]
Great release 🚀 👏🏼
